content/en/contribute.md

```diff
@@ -3,74 +3,121 @@ title: Contribute to NumPy
-
-We also have a bi-weekly _community call_, the details of which are announced on the mailing list. You are very welcome to join this call!
+The NumPy project welcomes your expertise and enthusiasm! Your
+choices aren't limited to programming -- in addition to
+
+- [Writing code](#writing-code)
+
+you can
+
+- [Review pull requests](#reviewing-pull-requests)
+- [Develop tutorials, presentations, and other educational material](#developing-educational-materials)
+- [Triage issues](#issue-triaging)
+- [Work on our website](#website-development)
+- [Contribute graphic design](#graphic-design)
+- [Translate website content](#translating-website-content)
+- Answer questions on [Gitter](https://gitter.im/numpy/numpy),
+  [Stack Overflow](https://stackoverflow.com/questions/tagged/numpy),
+  and [Reddit](https://www.reddit.com/r/Numpy/)
+- Participate in [mailing list](https://mail.python.org/mailman/listinfo/numpy-discussion) discussions
+- [Serve as a community coordinator](#community-coordination-and-outreach)
+- [Write grant proposals and help with other fundraising](#fundraising)
+
+If you're unsure where to start or how your skills fit in, _reach out!_ You
+can ask on the [mailing
+list](https://mail.python.org/mailman/listinfo/numpy-discussion) or
+[GitHub](http://github.com/numpy/numpy) (open an
+[issue](https://github.com/numpy/numpy/issues) or comment on a relevant
+issue).
+
+Those are our preferred channels (open source is open by nature), but
+if you prefer to talk privately, contact our community coordinators at
+`numpy-team@googlegroups.com` or on [Slack](https://numpy-team.slack.com)
+(write  `numpy-team@googlegroups.com` for an invite).
+
+We also have a biweekly _community call_, details of which are announced on
+the [mailing list](https://mail.python.org/mailman/listinfo/numpy-discussion).
+You are very welcome to join.
+
+Our community aspires to treat everyone equally and to value all
+contributions. We have a [Code of Conduct](/code-of-conduct) to foster an open
+and welcoming environment.
+
+### Writing code
+
+Programmers, this
+[guide](https://numpy.org/devdocs/dev/index.html#development-process-summary)
+explains how to contribute to the codebase.
+
+### Reviewing pull requests
+Even inexperienced contributors can help us deal with our surging PR count
```

Comment:
This message I'd rather tweak a bit. Inexperienced developers trying to move PRs forward often add more noise and overhead than being helpful.

Reply:
Indeed! Revised version clarifies the baseline skills.